site/_docs/history.md

```diff
@@ -29,9 +29,69 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 {% comment %}
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.31.0">1.31.0</a> / TODO
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.32.0">1.32.0</a> / TODO
+{: #v1-32-0}
+
+#### Breaking Changes
+{: #breaking-1-32-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 18;
+Guava versions 19.0 to 31.1-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-32-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-32-0}
+
+#### Build and test suite
+{: #build-1-32-0}
+
+#### Dependency version upgrade
+{: #dependency-1-32-0}
+
+#### Web site and documentation
+{: #site-1-32-0}
+{% endcomment %}
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.31.0">1.31.0</a> / 2022-07-24
 {: #v1-31-0}
 
+This release comes four months after [1.30.0](#v1-30-0),
+contains contributions from 30 authors, and resolves 81 issues.
```

Comment:
We'd need to add the "Among others, it is worth highlighting the following. ..." section.
IMHO the following items could be in the highlights section (among others):
- CALCITE-4865 Allow table functions to be polymorphic
- CALCITE-5107 Support SQL hint for `Filter`, `SetOp`, `Sort`, `Window`, `Values`